src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionItemExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Razor.Completion;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
+{
+    internal static class RazorCompletionItemExtensions
+    {
+        private readonly static string TagHelperAttributeCompletionDescriptionKey = "Razor.TagHelperAttributeDescription";
+        private readonly static string TagHelperElementCompletionDescriptionKey = "Razor.TagHelperElementDescription";
+
+        public static void SetTagHelperElementDescriptionInfo(this RazorCompletionItem completionItem, ElementDescriptionInfo elementDescriptionInfo)
+        {
+            if (completionItem is null)
```

Comment:
These can go away if it's nullable enable

Reply:
Can't because if other pieces of the code aren't `nullable enable` yet it wont do a runtime check 😄